.changeset/tall-llamas-crash.md

```diff
@@ -0,0 +1,7 @@
+---
+"@khanacademy/perseus": minor
+"@khanacademy/perseus-core": patch
```

Comment:
Isn't this a breaking change in `perseus-core`? If someone had defined a `PerseusRenderer` with `metadata` and upgraded `perseus-core` they would get a TS error and would be required to change their code?

Reply:
Yeah, I guess that's true!

Reply:
I've deprecated `metadata` rather than removing it outright, and changed all of these version bumps to `minor`.